code-challenges.js

```diff
@@ -21,6 +21,11 @@ const fruit4 = "kiwi"
 // --------------------1) Create a function that takes in a number and determines if the given number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below.
 
 // Pseudo code:
+// fucntion
+    // input: number
+    // below, at, or above boiling point
+// conditional statement
```

Comment:
Need some elaboration here: what are the conditions and what is the result of each condition? You used string interpolation in the function, put that in here too.